constantcontact.gemspec

```diff
@@ -23,9 +23,9 @@ Gem::Specification.new do |s|
   s.executables = []
   s.require_paths = [ "lib" ]
   s.test_files = Dir['spec/**/*_spec.rb']
-  
-  s.add_runtime_dependency("rest-client", '~> 1.6', '>= 1.6.7')
+
+  s.add_runtime_dependency("rest-client", '~> 1.8')
   s.add_runtime_dependency("json", '~> 1.8', '>= 1.8.1')
```

Comment:
I haven't had the chance to try it again since the last comment, but perhaps @pjg has more recent up-to-date experiences?